pages/wiki/backup.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Backup Wear OS
+layout: documentation
+---
+
+Before flashing AsteroidOS to your watch to replace WearOS, we advise to make a backup.
+
+# Manual backup method
+
+### 1. Temporarily install AsteroidOS
+
+Follow the install instructions for a temporary installation. Once booted into AsteroidOS, we can enable SSH or ADB with root access to all partitions on the internal storage.
+
+### 2. Enable SSH or ADB
+
+Open the settings app and the USB page. Select Developer mode to enable SSH. Or ADB mode to use ADB connection.
```

Comment:
It might be helpful to note that Developer mode is the default, so if AsteroidOS has just been booted for the first time, it's already configured for `ssh`.

Reply:
> I know it's still a draft, but maybe these comments are of some use to you.

Thats exactly why its a draft :D 
Was hoping for input. thank you.